src/com/exadel/bootcamp/CashMachines.java

```diff
@@ -1,22 +1,22 @@
 package com.exadel.bootcamp;
 
 import java.util.ArrayList;
+import java.util.List;
 
 public class CashMachines {
-    private static ArrayList<CashMachine> cashMachinesList = new ArrayList<>();
+    private static List<CashMachine> cashMachinesList = new ArrayList<>();
 
     public static void addCashMachineToList(CashMachine c) {
         cashMachinesList.add(c);
     }
 
     public static CashMachine getCashMachine(String name) {
-        CashMachine c = cashMachinesList.stream()
+        return cashMachinesList.stream()
```

Comment:
(cosmetic) A rule of thumb: if you start breaking a statement into separate lines, break *every* part. 
In this particular case,
```
.findFirst()
.orElse(null);
```